### PR TITLE
Add the Create WebP Image package

### PR DESCRIPTION
**Pull request by TomasBarry**
A redo of https://github.com/wbond/package_control_channel/pull/7867 since that one went stale.

The necessary updates have been made and I have verified that this package works.

### CODE REVIEW

**Comment by Ultra-Instinct-05** [comment]
A couple points, but otherwise seems fine.

1. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/.DS_Store - I suppose this is something MacOS specific and can be gitignored ?

2. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/CreateWebpImage.sublime-settings#L9 - 99% of the times, settings in Sublime Text Packages are snake_case by default. But whatever suits you.

3. https://github.com/TomasBarry/Sublime-create-webp-image/blob/master/CreateWebpImage.py#L89 - You use the `input` method only when you are implementing an `InputHandler`. Otherwise, it's not required.

**Review by Ultra-Instinct-05 — APPROVED** [review]

**Comment by TomasBarry** [comment]
Thanks for the approval @Ultra-Instinct-05. What needs to happen to get this merged in?

File: repository/c.json [hunk 1]
```diff
@@ -3937,6 +3937,19 @@
 				}
 			]
 		},
+		{
+			"author": ["Tomas Barry"],
```
Comment (braver) [inline repository/c.json:3941]: I didn't know we supported arrays here, also doesn't seem necessary in this case.

File: repository/c.json [hunk 2]
```diff
@@ -3937,6 +3937,19 @@
...
+			"name": "Create WebP Image",
+			"description": "Convert images to WebP from Sublime Text",
```
Comment (braver) [inline repository/c.json:3943]: You can put this in the GitHub repo description instead for ease of maintenance.

**Comment by braver** [comment]
I think you'd want to be more clear about the prerequisite. From experience I guess I need to install that utility in a way that puts cwebp in my $PATH, but for many users that's a few hurdles to many.

File: repository/c.json (hunk 1 above)
Comment (TomasBarry) [reply repository/c.json]: Addressed.

File: repository/c.json (hunk 2 above)
Comment (TomasBarry) [reply repository/c.json]: Addressed.

**Comment by TomasBarry** [comment]
@braver, I've addressed your comments and have included the latest changes as part of the release (a lapse on my part). The usage instructions have been updated to more explicitly reference the official documentation so that I always point to the latest instructions instead of potentially having stale instructions in my README.

Is there anything else I need to cover?

**Comment by braver** [comment]
Nope, all good 👍🏻

**Pull request merged**